src/main/java/cn/nukkit/event/player/PlayerChatEvent.java

```diff
@@ -21,25 +21,15 @@ public static HandlerList getHandlers() {
 
     protected Set<CommandSender> recipients = new HashSet<>();
 
-    public PlayerChatEvent(Player player, String message) {
-        this(player, message, "chat.type.text", null);
-    }
-
-    public PlayerChatEvent(Player player, String message, String format, Set<CommandSender> recipients) {
+    public void sendMessage(Player player, String message) {
         this.player = player;
         this.message = message;
+	this.format = "chat.type.text";
```

Comment:
this.format will be always overridden so even if you use setFormat() then sendMessage will be used after that and overridde changed format. So you should let as it is or check if this.format is empty then this.format = "chat.type.text"; otherwise let dev to use his format.